### ISSUE TITLE
connect srsRAN to commerical core

### ISSUE THREAD

**Issue by faratell** [srsran/srsRAN_4G#1397]
Hi.
I use srsRAN 4G with the B210 and Open5GS to create an open-source network, and it works well. Is it possible to use a commercial core (not an open-source core) alongside srsRAN 4G to create a 4G network?

**Comment by armin485** [comment]
I guess you should use core networks that support roaming such as Magma to roam your network data with comercial core networks.

**Comment by herlesupreeth** [comment]
> Is it possible to use a commercial core (not an open-source core) alongside srsRAN 4G to create a 4G network?

yes, as long as that commercial core has a 3GPP complaint S1AP and GTP-U interface implemented it should work with srsRAN 4G eNB
